babel.config.js

```diff
@@ -52,6 +52,29 @@ const config = {
 		isCalypsoClient && './inline-imports.js',
 	] ),
 	env: {
+		build_pot: {
+			plugins: [
+				[
+					'@wordpress/babel-plugin-makepot',
+					{
+						output: 'gutenberg-strings.pot',
```

Comment:
We should add this to `.gitignore`, and remove it as part of the `clean:build` script in `package.json`. Or alternatively just build it inside `build/i18n-calypso/`?